Dockerfile

```diff
@@ -41,6 +41,10 @@ COPY --from=builder /usr/local/lib/python3.8/site-packages/kapitan ./kapitan
 RUN ./kapitan/inputs/helm/build.sh \
  && ./kapitan/dependency_manager/helm/build.sh
 
+FROM golang:1.15 AS jsonnet-bundler
+
+RUN go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
```

Comment:
Let's download a specific version from the [releases](https://github.com/jsonnet-bundler/jsonnet-bundler/releases).